dokuwiki-text/src/main/java/org/xwiki/contrib/dokuwiki/text/internal/input/DokuWikiInputFilterStream.java

```diff
@@ -123,25 +123,42 @@ protected void read(Object filter, DokuWikiFilter proxyFilter) throws FilterExce
     {
         InputSource inputSource = this.properties.getSource();
         if (inputSource instanceof FileInputSource) {
-            try {
-                CompressorInputStream input = new CompressorStreamFactory()
-                        .createCompressorInputStream(new BufferedInputStream(
-                                ((InputStreamInputSource) inputSource).getInputStream()));
-                ArchiveInputStream archiveInputStream = new ArchiveStreamFactory()
-                        .createArchiveInputStream(new BufferedInputStream(input));
-                readDataStream(archiveInputStream, filter, proxyFilter);
-                input.close();
-            } catch (Exception e1) {
+            File f = ((FileInputSource) inputSource).getFile();
+            if (f.exists() && f.isDirectory()) {
```

Comment:
This if/else is not right, if exists() return false you should throw an error right away not try to open it as archive.